aws-sdk-core/lib/aws-sdk-core/api/docs/param_formatter.rb

```diff
@@ -44,7 +44,7 @@ def ref_value(ref, i, visited)
             end
           when ListShape then list(ref, i, visited)
           when MapShape then map(ref, i, visited)
-          when BooleanShape then "true"
+          when BooleanShape then "false"
```

Comment:
This is just for doc examples, right? And I assume this is to avoid the default presumption of examples being "accelerate on"?